app-misc/ddcutil/ddcutil-0.8.4.ebuild

```diff
@@ -53,6 +53,9 @@ src_prepare() {
 }
 
 src_configure() {
+	# Bug 607818.
+	replace-flags -O3 -O2
```

Comment:
I'll accept this for now, but can you please run UBSAN over the codebase? Perfectly valid code does not break with `-O3`

Reply:
I Can do that and send patch upstream. Sorry. Internet is dead and on mobile.